lib/district.rb

```diff
@@ -1,41 +1,18 @@
-require_relative 'enrollment'
-require 'bigdecimal'
+require 'pry'
+
 class District
   attr_reader :name
+  attr_accessor :enrollment_data
```

Comment:
The attr_accessor method convention allows us to automatically have getters/setters for class variables.